partnerships/community_success/hub-activity.md

```diff
@@ -0,0 +1,311 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "user_expressions": []}
+
+# Track Hub Usage with Grafana
+
++++ {"user_expressions": []}
+
+## Overview
+
+Grafana is an open source analytics and interactive visualization web application. Prometheus is an open-source monitoring and alerting platform that collects and stores metrics as time-series data, which feeds into Grafana as a data source.
+
+Grafana dashboard deployments for 2i2c hubs (k8s+JupyterHub) follow the templates outlined in the GitHub repository [jupyterhub/grafana-dashboards](https://github.com/jupyterhub/grafana-dashboards). Note that Prometheus data is retained for 1 year on 2i2c hubs.
+
++++ {"user_expressions": []}
+
+## Prerequisites
+
+### Create a Grafana service account and generate a token
+
+See [Grafana docs – Service Accounts](https://grafana.com/docs/grafana/latest/administration/service-accounts/) for more details.
+
+1. Navigate to the [2i2c Grafana](https://grafana.pilot.2i2c.cloud) website at `https://grafana.pilot.2i2c.cloud`.
+1. Open the {octicon}`three-bars` Menu and click on *{octicon}`gear` Administration > Users and access > Service accounts*.
+1. Click on the *Add service account* button on the top-right.
+1. Choose a descriptive *Display name*, e.g. `username-local-prometheus-access` and leave the role as *Viewer*. Click the *Create* button to confirm.
+1. You will see a new page with the details of the service account you have created. In the section *Tokens*, click the *Add service account token* button to generate a token to authenticate with the Grafana API.
+1. Choose a descriptive name for the token and then set a token expiry date. We recommend 1 month from now.
```

Comment:
worth noting that we'll then need to manually re-create the token whenever the expiry date hits, in a footnote or something?

Reply:
Footnote added in https://github.com/2i2c-org/team-compass/pull/859/commits/3e3f2d5be37317f986966db6a33b5a1457f82800.

There's a balance between accessibility and security to be struck here, what do we think? I also think that the Grafana token should be stored as an organizational 2i2c secret since there are multiple uses, to avoid secrets proliferating. Happy to update the team compass with what we decide on this.